client/sites/tools/controller.tsx

```diff
@@ -50,6 +51,11 @@ export function ToolsSidebar() {
 	);
 }
 
+export function advancedTools( context: PageJSContext, next: () => void ) {
+	context.primary = <HostingFeatures />;
```

Comment:
We also need to replace the translation of `hosting features` to `advanced tools` in the component, when `isEnabled( 'untangling/hosting-menu` )`.

Reply:
fixed in 4ae4e46fbedc68b0675700c8e7b0298126ad5cfd